android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -1069,7 +1072,10 @@ private void removeParticipantVideo(Participant participant, RemoteVideoTrackPub
     // ===== EVENTS TO RN ==========================================================================
 
     void pushEvent(View view, String name, WritableMap data) {
-        eventEmitter.receiveEvent(view.getId(), name, data);
+        ReactContext context= (ReactContext) view.getContext();
+        EventDispatcher eventDispatcher =
+                context.getNativeModule(UIManagerModule.class).getEventDispatcher();
+        eventDispatcher.dispatchEvent(new TwilioEvent(view.getId(),name,data));
```

Comment:
Can we add here a couple of lines pointing to the issue in the app https://github.com/senseobservationsystems/goalie-2-mobile-app/issues/3416 to let us know in the future why we changed this?
Maybe also in the new `TwilioEvent` class you created in `TwilioEvent.jaga` file.